tests/functional/Extension/Mention/MentionExtensionTest.php

```diff
@@ -58,8 +59,8 @@ public function testConfigStringGenerator(): void
         $environment->setConfig([
             'mentions' => [
                 'github_handle' => [
-                    'symbol'    => '@',
-                    'regex'     => '/^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}(?!\w)/',
+                    'prefix'    => '@',
+                    'regex'     => '[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}(?!\w)',
```

Comment:
Why are we still including a `prefix`? Shouldn't this just be part of the `regex` now?